Realm/RLMObjectBase_Private.h

```diff
@@ -26,6 +26,7 @@ NS_ASSUME_NONNULL_BEGIN
 @interface RLMObjectBase ()
 + (void)initializeLinkedObjectSchemas;
 + (bool)isEmbedded;
++ (bool)isDynamic;
```

Comment:
The thing this is used for is already covered by `shouldIncludeInDefaultSchema`.